### DISCUSSION TITLE
Bookmark History  please 😂🙏

### DISCUSSION THREAD

**Discussion by HypeNeutron** [alefragnani/vscode-bookmarks#403] [Ideas]
I have some problem when action on bookmark some time unguarded press delete bookmark on top it make bookmark all disappear. i think other people have this problem as well

**Comment by alefragnani** [comment]
Hi @RedNeutorn ,

Which are your expectations about a Bookmark History? I mean, how it would work?

* Would it allows you to revert the delete, like an `Undo` operation?
* Would it still display the content you had on that deleted line?
* Would it allows you to bring back that bookmarked line?

Could you provide more details?
